src/dmv.py

```diff
@@ -0,0 +1,99 @@
+import requests
+
+class CA_DMV():
+    API_ROOT = "https://www.dmv.ca.gov/wasapp/ipp2"
+    PATH_INIT = "initPers.do"
+    PATH_START = "startPers.do"
+    PATH_PROCESS = "processPers.do"
+    PATH_PLATE = "processConfigPlate.do"
+
+    START_DATA = {
+        "acknowledged": "true", 
+        "_acknowledged": "on",
+    }
+    PROCESS_DATA = {
+        "imageSelected": "none",
+        "vehicleType": "AUTO",
+        "licPlateReplaced": "GITHUB",
```

Comment:
I'm not sure about using the license plate of a random person. Should we prompt for this?

Reply:
I'm also not sure about it. As I mentioned in the description there are some other ways we can go about this (prompting would be one I didn't think of), but I don't think that doing this is *too* bad.

Note that license plate + VIN is something anyone can look up as its government collected data (hence why there are so many lookup sites). I picked this one the individual in question has an interesting license plate that users of this site are likely to have looked up, so I don't think I'd be giving them unwanted attention.

Reply:
Created #7 to track